packages/cashier/src/pages/payment-agent-transfer/payment-agent-transfer-form/payment-agent-transfer-form.jsx

```diff
@@ -19,9 +19,12 @@ const validateTransfer = (values, { balance, currency, transfer_limit }) => {
     const { is_ok, message } = validNumber(values.amount, {
         type: 'float',
         decimals: getDecimalPlaces(currency),
-        ...(transfer_limit.min && {
-            min: transfer_limit.min,
-            max: +balance >= transfer_limit.min && +balance < transfer_limit.max ? balance : transfer_limit.max,
+        ...(transfer_limit.min_withdrawal && {
+            min: transfer_limit.min_withdrawal,
+            max:
+                +balance >= transfer_limit.min_withdrawal && +balance < transfer_limit.max_withdrawal
+                    ? balance
+                    : transfer_limit.max_withdrawal,
```

Comment:
@george-usynin-binary Would be nice to extract this into a variable maybe.
It's out of your PR scope tho, You can ignore this comment 🙇🏻